template-files/android/AndroidManifest.xml

```diff
@@ -79,6 +79,15 @@
           android:host="expo.io"
           android:path="/expo-go"
           android:scheme="http"/>
+        
+        <data
+          android:host="expo.dev"
+          android:path="/expo-go"
+          android:scheme="https"/>
+        <data
+          android:host="expo.dev"
+          android:path="/expo-go"
+          android:scheme="http"/>
```

Comment:
Can remove this -- HTTP is not supported for .dev domains -- HSTS is enforced at the TLD level.

Reply:
Actually, sorry, I think we may want this. The reason is because this manifest is used to tell Android to open Expo Go when the user taps a link to http://expo.dev/expo-go. Even though browsers will immediately rewrite that URL to use HTTPS due to HSTS, the link is not sent through a browser first. It's handled by Android and we want it to open up Expo Go. Sorry for the confusion.